backend/migrations/2023-07-21-085933_gain_blossoms/up.sql

```diff
@@ -0,0 +1,28 @@
+-- Your SQL goes here
+CREATE TABLE guided_tours (
+    user_id UUID PRIMARY KEY,
+    editor_tour BOOLEAN NOT NULL DEFAULT false
+);
+
+CREATE TYPE track AS ENUM (
+    'Beginners Track',
+    'Seasonal Track',
+    'Completionist Track',
+    'Expert Track'
+);
+
+CREATE TABLE blossoms (
+    title TEXT PRIMARY KEY,
+    description TEXT,
+    track TRACK,
+    icon TEXT,
+    is_seasonal BOOLEAN NOT NULL
```

Comment:
Isn't this redundant to the track seasonal?

Reply:
I wanted to leave the possibility open to make a Blossom repeatable, even though it is not part of the Seasonal Track. I'm thinking for example Blossoms from the Expert Track, which are all centered around doing objectives that earn permacoins. Most of these could also be repeatable.